src/Microsoft.Data.SqlClient/tests/FunctionalTests/AlwaysEncryptedTests/ConnectionStringBuilderShould.cs

```diff
@@ -82,6 +145,23 @@ public void TestSqlConnectionStringBuilderColumnEncryptionSetting(SqlConnectionC
             VerifyColumnEncryptionSetting(connectionStringBuilder, sqlConnectionColumnEncryptionSetting == SqlConnectionColumnEncryptionSetting.Enabled);
         }
 
+        [Theory]
+        [InlineData(SqlConnectionAttestationProtocol.AAS)]
```

Comment:
I guess there can be a test case to pass `null` here?

Reply:
NotSpecified Value is added and test is passing.